src/modules/transformers.ts

```diff
@@ -0,0 +1,10 @@
+// transformer to convert string to decimal
```

Comment:
This whole file can be deleted now

Reply:
@yrobla can we drop it? or is still necessary?